website/src/controllers/payments/paypal.js

```diff
@@ -26,9 +26,10 @@ paypal.configure({
 });
 
 var parseErr = function(res, err){
-  //var error = err.response ? err.response.message || err.response.details[0].issue : err;
-  var error = JSON.stringify(err);
-  return res.json(400,{err:error});
+  var error = err.response ? err.response.error_description || err.response.error : 'Paypal returned an invalid error message.';
+  return res.json(400, {err: 'Paypal error: ' + error});
```

Comment:
Yeah, that was a bit awkward.  The idea was that that message was there in case Paypal responded in a weird way (didn't give an error or a normal response), but I agree that it looks a bit scary.

I'll implement those changes as soon as I've wrapped my brain around the test framework!